app/db/song.db.ts

```diff
@@ -24,8 +24,13 @@ export async function getSong({ id }: Pick<Song, "id">) {
   });
 }
 
-export async function getSongs() {
-  return await prisma.song.findMany();
+/**
+ * Gets the songs of the currently logged in user
+ * @param userId
+ * @returns
+ */
+export async function getUserSongs(userId: number) {
```

Comment:
Can we rename this to something like `getSongsByUser` or something to that affect? Would be nice to be able to type `getSong...` and have autocomplete fill for db functions relevant to the Song model